### ISSUE TITLE
Adicionar i18n para suportar multilanguage 🤝

### ISSUE THREAD

**Issue by ygcorrea** [feministech/feministech#40]

**Comment by ghost** [comment]
@ygcorrea teria interesse em trabalhar nessa issue aproveitando a Hacktoberfest?

**Comment by ygcorrea** [comment]
Oie, claro! pode deixar que faco (:

**Comment by ghost** [comment]
@ygcorrea vai precisar adicionar algum elemento visual novo no site? Ex.: botão de tradução ou mudança de idioma.

Se sim, poderia prototipar como ficaria no Figma? (Eu envio o link do modelo para edição)

**Comment by ygcorrea** [comment]
Sim, acho que vamos precisar de um dropdown pra realizar a troca de idiomas, se voce quiser, pode escolher um lugar pra colocar e me envia o prototipo de como vc acha que ficaria legal! Ai eu comeco a trabalhar em cima!
